modules/hydra-master-main.nix

```diff
@@ -79,30 +94,19 @@ in {
       <github_authorization>
         input-output-hk = ${builtins.readFile ../static/github_token}
       </github_authorization>
-      <githubstatus>
-        jobs = serokell:iohk-ops.*:required
-        inputs = jobsets
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
-      <githubstatus>
-        jobs = Cardano:cardano-ledger.*:required
-        inputs = chain
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
-      <githubstatus>
-        jobs = Cardano:cardano-ledger-specs.*:required
-        inputs = cardano-ledger-specs
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
-      <githubstatus>
-        jobs = serokell:cardano.*:required
-        inputs = cardano
-        excludeBuildFromContext = 1
-        useShortContext = 1
-      </githubstatus>
+
+      ${mkStatusBlocks [
+        { jobset = "iohk-ops"; inputs = "jobsets"; }
+        { jobset = "cardano-ledger"; inputs = "chain"; }
```

Comment:
btw, what in the role of the `inputs` attributes? (any why `chain`?)

Reply:
`inputs` is the name of the input from the hydra config, and i was just basing it on the old xml-ish config
but it looks like i need to include the rename from the other pr